### PR TITLE
TensorBoard 2.1.0

### PR DESCRIPTION
**Pull request by davidsoergel**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,8 +19,8 @@ env:
     - BAZEL=0.26.1
     - BAZEL_SHA256SUM=6c50e142a0a405d3d8598050d6c1b3920c8cdb82a7ffca6fc067cb474275148f
   matrix:
-    - TF_VERSION_ID=tensorflow==1.15.0rc3
-    - TF_VERSION_ID=tf-nightly==2.1.0.dev20191118
+    - TF_VERSION_ID=tensorflow==1.15.0
+    - TF_VERSION_ID=tf-nightly==2.1.0rc0
```
Comment (wchargin) [inline .travis.yml]: Should be `tensorflow==2.1.0rc0`; there’s no such `tf-nightly` version.

Comment (davidsoergel) [reply .travis.yml]: Whoops of course, thanks

File: RELEASE.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+  - Added ability to explore counterfactual examples for regression models (#2647)
+  - Added ability to consume arbitrary prediction-time information (#2660)
+  - Added ability to slice performance statistics by numeric features (in addition to categorical features) (#2678, #2704).
+  - Added PR/ROC curves by class for multi-class classification models (#2755)
+- Improvements for plugin developers:
+  - Added support for communication between TensorBoard and plugins in iframes (#2309, #2703)
+  - (Experimental) Added library for improved plugin integration (#2708)
+  - Enabled dynamic plugins in TensorBoard within Colab (#2798)
+- Security improvements, e.g. Content Security Policy configurations
+- Improved performance of various summary writers (#2899)  - thanks @hongjunChoi
```
Comment (nfelt) [inline RELEASE.md]: I'd maybe say "Reduced overhead of image, audio, and histogram summary writing API methods" (usually "summary writer" refers to writer objects, which are something else)

**Comment by davidsoergel** [comment]
Curious if I can revive this PR after the rebase.  If it doesn't work, I'll create a new one.

**Comment by davidsoergel** [comment]
Nope, that doesn't work.

**Comment by davidsoergel** [comment]
reopening while trying to figure this out...

**Comment by davidsoergel** [comment]
Abandoned in favor of #3002 (due to git rebase kerfuffle)

File: RELEASE.md (hunk 2 above)
Comment (davidsoergel) [reply RELEASE.md]: done

**Pull request closed without merging**